toolkits/springernature/packages/springer-nature-modal/README.md

```diff
@@ -0,0 +1,36 @@
+# Springer Nature Modal
+
+Display a modal (pop-up) window
+
+## Usage
+
+```html
+<button data-modal-for="example-modal">Click this button to open the modal</button>
+<div data-component-modal id="example-modal" class="c-modal js-hide" tabindex="0">
+    <div class="c-modal--content">
+        <h4 class="c-modal--title u-mb20">Modal Title</h4>
+        <div class="c-modal--body">
+            <p>This is the modal! It has a <a data-component-modal-close class="close-modal-link btn-close" href="">link</a> that can also close it.</p>
+        </div>
+        <button data-component="modal-close" class="c-modal--close btn-close link-like">&times;</button>
```

Comment:
Should this be data-component-modal-close

Reply:
Great catch!

Reply:
![hawkeyes](https://user-images.githubusercontent.com/11961647/73440455-3808dd00-4349-11ea-9014-367f3f8ec6c1.gif)